modules/publication/ajax/FileDelete.php

```diff
@@ -0,0 +1,52 @@
+<?php
```

Comment:
The AJAX directory is deprecated and should be replaced by proper PHP endpoints. Ideally there would be URLs like `/publications/files/$filename` which handle post/delete/get/put requests appropriately according to the HTTP standard.

I'm willing to punt on this and do it after the fact since this PR has been open for so long that we didn't have a proper router when it was first sent., but we really, really shouldn't be adding anything new that depends on the ajax directory.